docs/nodes/nodes-library/nodes/GoogleSheets/README.md

```diff
@@ -34,39 +34,137 @@ The final workflow should look like the following image.
+16. Toggle ***Keep Only Set*** to true.
+17. Click on ***Execute Node*** to run the node.
+:::
+
+![Using the Set node to increase the rent](./Set1_node.png)
+
+### 6. Google Sheets2 node (Update)
 
-### 4. Google Sheets1 node (Read)
+This node will update the rent with the new rent set in the previous node.
+
+1. Select the credentials that you entered in the previous Google Sheets node.
+2. Select 'OAuth2' in the ***Authentication*** field.
+3. Select 'Update' from the ***Operation*** dropdown list.
+4. In the ***Sheet ID*** field, enter the same string used in the previous Google Sheets node.
+5. In the ***Range*** field, enter the same range used in the previous Google Sheets node.
+6. Enter `ID` in the ***Key*** field.
+7. Click on the ***Add Option*** button and select 'Value Input Mode' from the dropdown list.
+8. Select 'User Entered' from the ***Value Input Mode*** dropdown list.
+9. Click on ***Execute Node*** to run the workflow.
+
+![Using the Google Sheets node to update the data to the Google Sheets spreadsheet](./GoogleSheets2_node.png)
+
+### 7. Google Sheets3 node (Read)
+
+This node will read the data from the Google Sheet.
 
 1. Select the credentials that you entered in the previous Google Sheets node.
 2. Select 'OAuth2' in the ***Authentication*** field.
 3. Select 'Read' from the ***Operation*** dropdown list.
 4. In the ***Sheet ID*** field, enter the same string used in the previous Google Sheets node.
 5. In the ***Range*** field, enter the same range used in the previous Google Sheets node.
-6. Click on ***Execute Node*** to run the workflow.
+6. Click on the ***Add Option*** button and select 'Value Render Mode' from the dropdown list.
+7. Select 'Formatted Value' from the ***Value Render Mode*** dropdown list.
+8. Click on ***Execute Node*** to run the workflow.
+
+![Using the Google Sheets node to read data from the Google Sheets spreadsheet](./GoogleSheets3_node.png)
+
+## FAQ
+
+### How to read the data in the format they were entered in Google Sheets?
+
+By default, the unformatted values are read. For example, if your sheets have a column which contains dates (2020-09-24) the node will read them as numbers, perform the mathematical operation (subtraction for this example), and will give you the output (output for the example of the date: 1987).
+
+If you want the formatted values, click on ***Add Option*** and select 'Formatted Value'. This will return the values as they are. For example, the dates will be returned in the format they were entered (2020-09-24).
+
+If you want to return the formula used in the cells, click on ***Add Option*** and select 'Formula'.
+
+
+### How to append an array in the Google Sheets?
+
+To insert the data in Google Sheets, you have to first convert the data in a valid JSON (key, value) format. You can use the [Funtion node](../../core-nodes/Function/README.md) to convert the array into JSON format.
+:::
+
+### How to return all the values with the Lookup operation?
```

Comment:
Feel free to point to these FAQs for some of these explanations mentioned above.